hedera-mirror-web3/src/main/java/com/hedera/services/utils/EntityIdUtils.java

```diff
@@ -214,4 +216,8 @@ private static long[] parseLongTriple(final String dotDelimited) {
     public static String asHexedEvmAddress(final Id id) {
         return CommonUtils.hex(asEvmAddress((int) id.shard(), id.realm(), id.num()));
     }
+
+    public static EntityId entityIdFromId(Id id, EntityType type) {
+        return new EntityId(id.shard(), id.realm(), id.num(), type);
+    }
```

Comment:
Might need to check for null. Also you can omit the `EntityType` parameter if you like to simplify it. It's not really used. Can just put UNKNOWN internally.
```suggestion
    public static EntityId entityIdFromId(Id id) {
        if (id == null) {
            return null;
        }
        return new EntityId(id.shard(), id.realm(), id.num(), EntityType.UNKNOWN);
    }
```

Reply:
Updated.